Package.resolved

```diff
@@ -0,0 +1,34 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "IDZSwiftCommonCrypto",
+        "repositoryURL": "https://github.com/iosdevzone/IDZSwiftCommonCrypto",
+        "state": {
+          "branch": null,
+          "revision": "d824371e670bd57eb456bbc41139b4997f7207b8",
+          "version": "0.13.1"
+        }
+      },
+      {
+        "package": "SwiftBytes",
+        "repositoryURL": "https://github.com/skogland/swift-bytes.git",
```

Comment:
The swift-bytes repo needs a version tag on the release with SPM-support to be included. This points to my fork with a version tag. Update the URL when the master is tagged.